explainpowershell.analysisservice/MetaData.cs

```diff
@@ -7,11 +7,12 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 using System.Text.Json;
-using Microsoft.Azure.Cosmos.Table;
 using explainpowershell.models;
 using System.Linq;
 using System.Collections.Generic;
 using System.Reflection;
+using Azure.Data.Tables;
+using Azure.Data.Tables.Sas;
```

Comment:
Are you sure we need this? I don't think we do.

Reply:
We don't need Sas, forgot to remove it